dataland-e2etests/src/test/kotlin/org/dataland/e2etests/Constants.kt

```diff
@@ -43,6 +43,7 @@ val FRONTEND_DISPLAYED_FRAMEWORKS = listOf(
     DataTypeEnum.lksg,
     DataTypeEnum.sfdr,
     DataTypeEnum.p2p,
+    DataTypeEnum.sme,
```

Comment:
Wurde wahrscheinlich bei der sme Story vergessen. Hätte das nicht in einem Test auffallen müssen?

Reply:
Dazu muss ich nochmal in die Tests reinschauen. Sage dir dann hier Bescheid was rauskam

Reply:
Das fällt normalerweise in einem Test auf, der failt.
Da das aber auch im feature selbst vergessen wurde in der Sme-Story, merkt der Test das nicht:

![image](https://github.com/d-fine/Dataland/assets/99255296/6569f979-09cc-4004-9680-fc6c5f77d03c)

Reply:
Das hängt mit dem Change zusammen, den du am Telefon besprechen möchtest in der anderen conversation.